src/ryu_faucet/org/onfsdn/faucet/valve.py

```diff
@@ -618,6 +627,8 @@ def port_delete(self, dp_id, port_num):
 
         # delete all rules matching this port in all tables.
         for table in self.all_valve_tables():
+            #if table == self.dp.acl_table:
+	    #  continue
```

Comment:
You are right -- it shouldn't be commented. We don't want to drop ACL rules unless the user requests the change.